schema/tmpBuilders.go

```diff
@@ -139,14 +139,23 @@ func (ts *TypeSystem) Init() {
 }
 func (ts *TypeSystem) Accumulate(typ Type) {
 	typ._Type(ts)
-	ts.namedTypes[typ.Name()] = typ
+	name := typ.Name()
+	if _, ok := ts.namedTypes[name]; ok {
+		// Should we error or panic here?
+		// We'll overwrite a type, and end up with duplicate names.
+	}
```

Comment:
I think a panic is fine here.  This is not an interface that we're expecting to be heavily directly used by the time we've finished all the parsing features we want to have.

Reply:
alright, will do